README.md

```diff
@@ -1,25 +1,25 @@
 # FIWARE IoT Agent Framework
 
 [![License badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
-[![Documentation badge](https://readthedocs.org/projects/fiware-iotagent-node-lib/badge/?version=latest)](http://fiware-iotagent-ul.readthedocs.org/en/latest/?badge=latest)
+[![Documentation badge](https://readthedocs.org/projects/iotagent-node-lib/badge/?version=latest)](http://iotagent-node-lib.readthedocs.org/en/latest/?badge=latest)
 [![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware)
 
 ## Index
 
 * [Overview](#overview)
-* [Build & Install](#buildinstall)
-* [API Overview](#apioverview)
-  * [About API](#aboutapi)
-  * [Device Provisioning API](#provisioningapi)
-  * [Configuration API](#configurationapi)
-* [Advanced Topics](#advancedtopics)
-  * [Secured access to the Context Broker](#securedaccess)
-  * [Data mapping plugins](#datamapping)
-  * [Old IoTAgent data migration](#datamigration)
-* [Testing](#librarytesting)
-* [Development Documentation](#development)
-
-## <a name="overview"/> Overview
+* [Build & Install](#build--install)
```

Comment:
If at the end

```
##  Build and Install
```

is used, then I undertand in the index we should have:

```
* [Build and Install](#build-and-install)
```

Reply:
Fixed in c80883

Reply:
Actually  I  just regenerated the whole ToC using markdown-toc instead.  fixed in ad029d4
This will ensure all ToC links are valid without making manual edits.